src/components/TokenCard/TokenCard.tsx

```diff
@@ -117,22 +117,9 @@ export default function TokenCard({ showRedirectModal }: Props) {
 
   const poolsIncentivizedLabel = useMemo(() => {
     const sparkle = <img src={sparkleIcon} alt="sparkle" className="relative -top-2 -mr-10 inline h-10 align-top" />;
-    const arbitrumLink = <ExternalLink href={getIncentivesV2Url(ARBITRUM)}>Arbitrum</ExternalLink>;
     const avalancheLink = <ExternalLink href={getIncentivesV2Url(AVALANCHE)}>Avalanche</ExternalLink>;
-    if (arbitrumIncentiveState?.lp?.isActive && avalancheIncentiveState?.lp?.isActive) {
-      return (
-        <Trans>
-          {arbitrumLink} and {avalancheLink} GM Pools are{" "}
-          <span className="whitespace-nowrap">incentivized{sparkle}.</span>
-        </Trans>
-      );
-    } else if (arbitrumIncentiveState?.lp?.isActive) {
-      return (
-        <Trans>
-          {arbitrumLink} GM Pools are <span className="whitespace-nowrap">incentivized{sparkle}.</span>
-        </Trans>
-      );
-    } else if (avalancheIncentiveState?.lp?.isActive) {
+
+    if (avalancheIncentiveState?.lp?.isActive) {
```

Comment:
what's the point of removing logic that should be turned off by keeper config?

Reply:
but arbitrum will be active. but with just one pool. and we want to not take it into account